src/native/corehost/test/mockhostfxr/mockhostfxr.cpp

```diff
@@ -20,7 +20,7 @@ SHARED_API int HOSTFXR_CALLTYPE hostfxr_main_startupinfo(const int argc, const p
 
     const pal::string_t dotnet_folder = get_directory(dotnet_root);
```

Comment:
`dotnet_root` should already be the folder. I guess that would explain why the tests have been working without `== 0 ` in the condition 😬